src/CookieConsent.tsx

```diff
@@ -54,15 +56,18 @@ const CookieConsentProvider: React.FC<React.PropsWithChildren<Props>> = ({
       const oneTrustActiveGroups = (
         window.OnetrustActiveGroups?.split(',') as Category[]
       )?.filter(Boolean);
+      const initialConsent =
+        oneTrustActiveGroups && oneTrustActiveGroups.length
```

Comment:
`oneTrustActiveGroups?.length` -> maybe just for possible null value of oneTrustActiveGroups

Reply:
it is already being checked, not sure I understand :D

Reply:
ignore me for this one...my oversight xD